### DISCUSSION TITLE
I get black screen on Catalina with geforce 210 - What can i try?

### DISCUSSION THREAD

**Discussion by cipriano200** [chris1111/Legacy-Video-patch#44] [Q&A]
Maybe i am an idiot, but didnt checked if sip is disabled.

Maybe is that the issue?

I am new to all this, still learning.

Thanks

**Comment by chris1111** [comment]
SIP is disable by the bootloader

**Reply by cipriano200** [reply]
Ok

What you think i should try now?

**Reply by chris1111** [reply]
Follow the Topic I just test New Install Catalina 10.15.7(19H1030) Succes

See Video Succces Install ➤ [Video test GeForce 210 Catalina 10.15.7(19H1030)](https://youtu.be/HIIGmZV_-Uc)

**Comment by cipriano200** [comment]
It works now

I updated catalina to version 10.15.7(19H1030) but was still not working with the patch, gave me black screen, so i figured is not the issue from patch.

From mistake i checked some options on clover options before booting:
- inject nvidia
- use nvidia drivers
- load video bios

I updated them using clover configuration after that. Not sure if all 3 options are needed.

And it worked. I get 1920x1080 resolution.

Thank you Chris for the patch

**Reply by sherazh1414** [reply]
patch not working for my hackintosh 10.15.7 GPU is GT 220 1GB RAM. Donot know the reason getting blank screen after chris patch. "IOconsoleUsers:GIOScreenLockState 3". Help!

**Comment by chris1111** [comment]
Only this option is need
inject nvidia
Please post the result here 
https://www.tonymacx86.com/threads/full-acceleration-for-intel-hd-3000-and-older-nvidia-graphics-in-macos-catalina.290488/page-17
then tell us to using latest update from here
https://github.com/chris1111/Legacy-Video-patch

**Reply by cipriano200** [reply]
Ok

Will do that today

**Reply by chris1111** [reply]
Sadly tonymacs leaves a package that is out of date for the latest Catalina and users continue to use it and their hackintosh becomes unusable

**Comment by JPaulMora** [comment]
Have you tried this GPU with Big Sur?
